.venv/.Python

```diff
@@ -0,0 +1 @@
+/System/Library/Frameworks/Python.framework/Versions/2.7/Python
```

Comment:
Is this related only for macos?

Reply:
doh! my bad 😞 
I added it unintentionally